your_daily_dose_malware/main.py

```diff
@@ -1,12 +1,31 @@
 import click
 from commands.scraper import scraper
+import requests
+from dotenv import load_dotenv
+import os
 
+load_dotenv()
+@click.command(help="dowloading malwares in zip file  by using api from 'https://mb-api.abuse.ch/api/v1/' then write t")
+@click.argument('sha256_names', type=click.File('r'),)
+def run_scrapper(sha256_names):
```

Comment:
Shall we move this under `your_daily_dose_malware/commands/scraper.py` so `main` would be entry point and sub commands would be part of commands folder?

Reply:
Yes I thinks Is better like this